kubejs/server_scripts/enigmatica/kubejs/base/loot_tables/repurposed_structures/mansion.js

```diff
@@ -0,0 +1,179 @@
+onEvent('generic.loot_tables', (event) => {
+    const pools = [
+        {
+            rolls: { min: 8, max: 16 },
+            entries: [
+                {
+                    item: 'farmersdelight:noodle_soup',
+                    weight: 50,
+                    count: [4, 16]
+                },
+                {
+                    item: 'ars_nouveau:source_berry_pie',
+                    weight: 50,
+                    count: [4, 8]
+                },
+                {
+                    item: 'alexsmobs:kangaroo_burger',
+                    weight: 50,
+                    count: [4, 8]
+                },
+                {
+                    item: 'undergarden:dweller_steak',
+                    weight: 50,
+                    count: [2, 8]
+                },
+                {
+                    item: 'simpledelights:orange_chicken',
+                    weight: 50,
+                    count: [2, 8]
+                },
+                {
+                    item: 'immersivecooking:pyttipanna',
+                    weight: 50,
+                    count: [2, 8]
+                },
+                {
+                    item: 'farmersdelight:apple_cider',
+                    weight: 50,
+                    count: [2, 8]
+                },
+                {
+                    item: 'simplefarming:sake',
+                    weight: 40,
+                    count: [1, 2]
+                },
+                {
+                    item: 'sushigocrafting:tobiko_tuna_cucumber_california',
```

Comment:
`Item.of('sushigocrafting:tobiko_tuna_cucumber_california', '{Weights:[I;0,0,0,0,0,0],Spices:{}}')`